Doc/howto/free-threading-python.rst

```diff
@@ -1,18 +1,18 @@
 .. _freethreading-python-howto:
 
-**********************************************
-Python experimental support for free threading
-**********************************************
+*********************************
+Python support for free threading
+*********************************
 
-Starting with the 3.13 release, CPython has experimental support for a build of
+Starting with the 3.13 release, CPython has support for a build of
 Python called :term:`free threading` where the :term:`global interpreter lock`
 (GIL) is disabled.  Free-threaded execution allows for full utilization of the
 available processing power by running threads in parallel on available CPU cores.
 While not all software will benefit from this automatically, programs
 designed with threading in mind will run faster on multi-core hardware.
 
-**The free-threaded mode is experimental** and work is ongoing to improve it:
-expect some bugs and a substantial single-threaded performance hit.
+The free-threaded mode is working and continues to be improved, but
+expect some bugs and a substantial performance hit in single-threaded workloads.
```

Comment:
The situation is much improved in 3.14 compared to 3.13, delete "substantial"?

```suggestion
The free-threaded mode is working and continues to be improved, but
expect some bugs and a performance hit in single-threaded workloads.
```

Or shall we can reword this?

@Yhg1s @colesbury Any suggestions?